### PR TITLE
Moving filesstore services into shared folder

### PR DESCRIPTION
**Pull request by jespino**
#### Summary
Moving filesstore services into shared folder

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-33323

#### Related PR
enterprise: mattermost/enterprise#925

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: app/file.go [hunk 1]
```diff
@@ -36,7 +36,7 @@ import (
 	"github.com/mattermost/mattermost-server/v5/mlog"
 	"github.com/mattermost/mattermost-server/v5/model"
 	"github.com/mattermost/mattermost-server/v5/plugin"
-	"github.com/mattermost/mattermost-server/v5/services/filesstore"
+	"github.com/mattermost/mattermost-server/v5/shared/filesstore"
```
Comment (agnivade) [inline app/file.go:39]: Just a small nit from my side: can we use this opportunity to rename it to `filestore` rather than `filesstore`?

Comment (jespino) [reply app/file.go]: Done.

**Review by agnivade — APPROVED** [review]

**Review by nronas — APPROVED** [review]
LGTM

**Review by mgdelacroix — APPROVED** [review]

**Comment by jespino** [comment]
@jupenur @srkgupta I haven't change the code itself, but because I'm moving it, it is complaining. I don't know if is needed to review that, or we want to change something before keep going.

**Comment by jupenur** [comment]
Thanks for the ping @jespino. Dismissed the alerts, should be good now.

**Review by lindy65 — APPROVED** [review]
Thanks @jespino - looks good to me 👍 Tested as per test steps on Jira ticket and didn't find any issues...

**Pull request merged**